drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/test/java/org/drools/workbench/screens/dtablexls/backend/server/conversion/GuidedDecisionTableGeneratorListenerTest.java

```diff
@@ -1729,6 +1729,165 @@ public void testMissingTemplateKeyValues() {
 
     @Test
     //https://issues.jboss.org/browse/GUVNOR-2030
+    public void testMissingTemplateKeyValues2() {
```

Comment:
What's the difference between ```testMissingTemplateKeyValues2``` and ```testMissingTemplateKeyValues```. A more descriptive test name would help here. I find it hard to understand what these tests do. I don't know much about the feature and used APIs so that contributes to that, of course ;).